src/mito2/src/manifest/storage.rs

```diff
@@ -424,6 +461,69 @@ impl ManifestObjectStore {
     pub async fn read_file(&self, path: &str) -> Result<Vec<u8>> {
         self.object_store.read(path).await.context(OpenDalSnafu)
     }
+
+    /// Get the size(Byte) of the delta file by delta version.
+    /// If the delta file does not exist, return None.
+    pub fn delta_file_size(&self, version: ManifestVersion) -> Option<u64> {
+        self.manifest_size_map.get(&delta_file(version)).copied()
+    }
+
+    /// Get the size(Byte) of the checkpoint file by checkpoint version.
+    /// If the checkpoint file does not exist, return None.
+    pub fn checkpoint_file_size(&self, version: ManifestVersion) -> Option<u64> {
+        self.manifest_size_map
+            .get(&checkpoint_file(version))
+            .copied()
+    }
+
+    /// Compute the size(Byte) in manifest size map.
+    pub fn get_total_manifest_size(&self) -> u64 {
```

Comment:
style(nitpick): Remove `get_` and use the same method name as the manager.
```suggestion
    pub fn total_manifest_size(&self) -> u64 {
```